### PR TITLE
Update guide for clarity.

### PR DESCRIPTION
**Pull request by jordanmack**
This revised guide includes changes for clarity, and contains all the discussed updates from the GitHub conversation page.

All the details should be the same with one exception which should be reviewed. In "RBF Check Rules" I have changed the way the five rules are grouped to make it easier to understand. There are no functional changes, but I would suggest updating the comments in `pool.rs` to match.

### CODE REVIEW

File: website/docs/essays/tx-pool-rbf.md [hunk 1]
```diff
@@ -9,68 +9,107 @@ import Link from "@docusaurus/Link";
...
 
-<img src={useBaseUrl("img/rbf.png")} width="70%" height="70%"/>
+The strategy used to replace an existing transaction in the tx-pool with a higher fee one is known as Replace-By-Fee (RBF). This was first introduced by Bitcoin in [BIP 125](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki).
 
-In the above example, the transaction `tx-a` is already in tx-pool and the new `tx-b` has a conflicted input `input1` with `tx-a`, so `tx-a` will be replaced by `tx-b` in the tx-pool if all RBF rules are satisfied.
+To use RBF to replace an existing transaction in the tx-pool with a new one, the new transaction must share at least one input with the old transaction and include a sufficiently high RBF fee. (RBF fees are outlined below.) All other inputs and outputs in the transaction can be different. The new transaction will replace the old transaction in the tx-pool, and the old transaction will be removed from tx-pool with status "Rejected".
```
Comment (chenyukang) [inline website/docs/essays/tx-pool-rbf.md:21]: maybe better?
```
and the old transaction with its descendants will be 
```

File: website/docs/essays/tx-pool-rbf.md [hunk 2]
```diff
@@ -9,68 +9,107 @@ import Link from "@docusaurus/Link";
...
+## Why Use RBF?
 
-<img src={useBaseUrl("img/rbf-merge.png")} width="70%" height="70%"/>
+RBF allows for adjustment of pending transactions and can be beneficial in several scenarios. RBF allows pending transactions to:
 
-For instance, the above transaction `tx-a`, `tx-b` and their descendants are already in the tx-pool, a new transaction `tx-e` has conflicted inputs with `tx-a` and `tx-b`, so `tx-a`,`tx-b`, `tx-c`, `tx-d` are all replaced by `tx-e` when RBF happens.
+- **Accelerate Transaction Confirmation**: RBF allows transactions that are not confirming fast enough to increase their fee to get higher prioritization. This can happen if an incorrect fee was paid or if network conditions change suddenly, resulting in a fee that is too low under current conditions.
+- **Transaction Cancellation**: RBF enables the cancellation of a pending transaction. This is done by submitting a new transaction that consumes the same inputs as the old transaction but redirects the funds back to the sender instead of the original recipient.
```
Comment (chenyukang) [inline website/docs/essays/tx-pool-rbf.md:27]: oh, this is a proper usage!

File: website/docs/essays/tx-pool-rbf.md [hunk 3]
```diff
@@ -9,68 +9,107 @@ import Link from "@docusaurus/Link";
...
-To simplify the use model, we add a new field `min_replace_fee` in the result of `get_transaction`, indicating the minimal fee to replace the transaction:
+`sum(replaced_tx_fee)` is the sum of the fees from all old transactions that are being directly replaced by the new transaction.
+
+`(min_rbf_rate * new_tx_size)` is the `min_rbf_rate` from `ckb.toml` and it is expressed in Shannons/KB. This is multiplied by the size of the new transaction.
+
+`min_replace_fee` is the total of all the fees being removed from the tx-pool, plus the new transaction calculated at the RBF rate. A single new transaction can replace multiple old transactions in the tx-pool, so it only makes sense that the replacement new transaction offers more fees than those it is replacing.
+
+### Using RBF via the CKB RPC
+
+The `min_replace_fee` field has been added to the result of `get_transaction` as a simple way to get the required fees to replace a single transaction with one of am identical size.
```
Comment (chenyukang) [inline website/docs/essays/tx-pool-rbf.md]:
```suggestion
The `min_replace_fee` field has been added to the result of `get_transaction` as a simple way to get the required fees to replace a single transaction with one of an identical size.
```

File: website/docs/essays/tx-pool-rbf.md (hunk 1 above)
Comment (jordanmack) [reply website/docs/essays/tx-pool-rbf.md:21]: Added.

File: website/docs/essays/tx-pool-rbf.md (hunk 3 above)
Comment (jordanmack) [reply website/docs/essays/tx-pool-rbf.md]: Updated.

**Pull request merged**